### PR TITLE
sorted module list ; improved release_planning.rb (search bar, sorted modules by number of commits, downloads and name)

### PR DESCRIPTION
**Pull request by daianamezdrea**

### CODE REVIEW

File: daily_open_prs.rb [hunk 1]
```diff
@@ -110,7 +110,7 @@
 end
 
 #Creates the CSV files
-CSV.open("daily_open_prs.csv", "w") do |csv|
+CSV.open("pr_work_done.csv", "w") do |csv|
```
Comment (tphoney) [inline daily_open_prs.rb:113]: this look strange

File: release_planning.rb [hunk 2]
```diff
@@ -2,12 +2,14 @@
 
 require 'optparse'
 require_relative 'octokit_utils'
+require_relative 'release_planning/pupet_module'
```
Comment (tphoney) [inline release_planning.rb:5]: sorry to be a debbie downer, the javascript plugins will not work in jenkins. due to the over zealous lock down there.

File: release_planning/pupet_module.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+class PuppetModule
```
Comment (tphoney) [inline release_planning/pupet_module.rb:2]: this is not used by anything outside of the release scripts. it should just live in the release planning script

File: release_planning/puppet_module_augmenter.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+class PuppetModuleAugmenter
```
Comment (tphoney) [inline release_planning/puppet_module_augmenter.rb:1]: this is not used by anything outside of the release scripts. it should just live in the release planning script

**Review by tphoney — CHANGES_REQUESTED** [review]
There are a few things to clean up first

**Comment by daianamezdrea** [comment]
replaced by https://github.com/underscorgan/community_management/pull/68

**Pull request closed without merging**